### PR TITLE
Better documentation for MELPA Stable

### PR DESCRIPTION
**Pull request by dunn**
I'm not wedded to any of the particular wording or organization choices here, but I think (as you know :see_no_evil: #3306) that Stable could be featured a bit more prominently.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,17 +51,42 @@ Enable installation of packages from MELPA by adding an entry to
...
-If your Emacs has the variable `package-pinned-packages`, you can
-customize or modify that variable as needed. Otherwise, use the
-separate
-[package-filter.el](https://github.com/milkypostman/package-filter)
-package which we provide.
+* If your Emacs has the variable `package-pinned-packages` (available
+  in 24.4 and later), you can customize or modify that variable as
+  needed.
 
+* You can use the
```
Comment (vyp) [inline README.md]: Shouldn't this still be "Otherwise, you can use..." as before?

Comment (dunn) [inline README.md]: There's a third option (Stable) so that didn't seem appropriate, to me at least.

Comment (vyp) [inline README.md]: Ah I missed that this is a list! So yes it makes sense, sorry.

**Comment by AdrieanKhisbe** [comment]
:+1:  :)

File: README.md [hunk 2]
```diff
@@ -51,17 +51,42 @@ Enable installation of packages from MELPA by adding an entry to
...
+
+### MELPA Stable
+
+By default, MELPA provides the very latest revisions of packages.  If
+you prefer to only receive updates for tagged releases, use
+[https://stable.melpa.org](MELPA Stable) instead:
+
+```lisp
+(add-to-list 'package-archives
+             '("melpa" . "https://stable.melpa.org/packages/") t)
```
Comment (milkypostman) [inline README.md]: I think the name of this archive should be "melpa-stable"

File: README.md [hunk 3]
```diff
@@ -148,6 +176,11 @@ through this process.
   recipe buffer: this will also prompt you to install the
   freshly-built package.
 
+  If the package has tagged releases, confirm that the correct version is
+  detected by running `STABLE=t make recipes/<NAME>`.  The version
```
Comment (milkypostman) [inline README.md]: Suggested revision,

> If the repo contains tags for releases, confirm the correct stable version is built by running ...

File: README.md (hunk 2 above)
Comment (milkypostman) [inline README.md]: this would match with the html page as well.

**Pull request merged**